components/SiteMenu.vue

```diff
@@ -53,74 +53,58 @@ function handleLinkClick () {
 
 const items = [
   {
-    label: 'Mint NFT',
+    label: 'NFT Book',
```

Comment:
```suggestion
    label: 'Publish',
```

Reply:
I would argue in book press they would already know something about nft or wallet or blockchain, they need LIKE

Reply:
no strong opinion